chpl/chpl-auth/src/main/java/gov/healthit/chpl/auth/EmailBuilder.java

```diff
@@ -73,6 +74,12 @@ public EmailBuilder recipients(final List<String> addresses) {
      */
     public EmailBuilder subject(final String val) {
         subject = val;
+        //Add the environment to the subject
+        String suffix = "";
+        if (!StringUtils.isEmpty(System.getenv("emailSubjectSuffix"))) {
+            suffix = " [" + System.getenv("emailSubjectSuffix") + "]";
+            subject = subject + " " + suffix;
```

Comment:
I think this'll add two spaces after the previous subject and before the `[`

Reply:
Appreciate the removal of the extra space, but I would like to have those `'[` and `]` back. Probably not required, just a desire

Reply:
Oh, never mind. I see it in the properties. This works